<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>菜单</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>741</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTabBar::tab{width:150}
QTabBar::tab{height:50}</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_rc">
    <attribute name="title">
     <string>热菜</string>
    </attribute>
    <widget class="QLabel" name="hsr">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>151</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="hsr_label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>红烧肉    32￥</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_hsr_add">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_hsr_sub">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="hsr_num">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="ssgy">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>0</y>
       <width>131</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="ssgy_lable">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>30</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>松鼠鳜鱼    68￥</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ssgy_sub">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>70</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="ssgy_num">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>70</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_ssgy_add">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>70</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLabel" name="yxrs">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>151</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="yxrs_label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>160</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>鱼香肉丝   36￥</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_yxrs_add">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>200</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_yxrs_sub">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>200</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="yxrs_num">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>200</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="qc">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>130</y>
       <width>131</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="qc_label">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>160</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>青菜    16￥</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_qc_add">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>200</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_qc_sub">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>200</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="qc_num">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>200</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_lc">
    <attribute name="title">
     <string>凉菜</string>
    </attribute>
    <widget class="QLabel" name="phg">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>151</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="md">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>0</y>
       <width>131</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="phg_label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>拍黄瓜    12￥</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_phg_add">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>80</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLabel" name="phg_num">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>80</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_phg_sub">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>80</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="md_label">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>毛豆   16￥</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_md_add">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>70</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLabel" name="md_num">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>70</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_md_sub">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>70</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="hsm">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>151</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="hsm_label">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>170</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>花生米    8￥</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_hsm_add">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>220</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLabel" name="hsm_num">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>220</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_hsm_sub">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>220</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="pd">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>130</y>
       <width>141</width>
       <height>121</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="pd_label">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>170</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>皮蛋   18￥</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_pd_add">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>220</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLabel" name="pd_num">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>220</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_pd_sub">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>220</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_yl">
    <attribute name="title">
     <string>饮料</string>
    </attribute>
    <widget class="QLabel" name="kl">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>141</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="kl_lable">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>30</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>可乐    3￥</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_kl_add">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLabel" name="kl_num">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>70</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_kl_sub">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>70</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QLabel" name="xb">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>0</y>
       <width>131</width>
       <height>111</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="xb_lable_">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>雪碧    3￥</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_xb_add">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>70</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QLabel" name="xb_num">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>70</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_xb_sub">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>70</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="zhuohao">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>桌号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="zongjia">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>41</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>总价：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="order_info">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>320</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>订单信息</string>
   </property>
  </widget>
  <widget class="QPushButton" name="send_order">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>320</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>提交订单</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>320</y>
     <width>89</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QLabel" name="labeltable">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>330</y>
     <width>21</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelprice">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>330</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
